plugins/repositories/core/pom.xml

```diff
@@ -33,34 +32,9 @@
 
   <dependencies>
     <dependency>
-      <groupId>com.fasterxml.jackson.core</groupId>
-      <artifactId>jackson-core</artifactId>
-      <version>${fasterxml.jackson.version}</version>
-    </dependency>
-    <dependency>
-      <groupId>com.fasterxml.jackson.core</groupId>
-      <artifactId>jackson-annotations</artifactId>
-      <version>${fasterxml.jackson.version}</version>
-    </dependency>
-    <dependency>
-      <groupId>com.fasterxml.jackson.core</groupId>
-      <artifactId>jackson-databind</artifactId>
-      <version>${fasterxml.jackson.version}</version>
-    </dependency>
-    <dependency>
+      <!-- this is required by OSGi bean declared in src/main/resources-filtered/OSGI-INF/blueprint/beans.xml -->
```

Comment:
@graimundo, is this OK?

Reply:
`jackson-jaxrs-json-provider` will also bring `jackson-jaxrs-base` and `jackson-module-jaxb-annotations` as transitive dependencies. As far as I can see, modules from `com.fasterxml.jackson.core` are not required.